front-end/src/app/cruds/pacientes/paciente-listar/paciente-listar.component.ts

```diff
@@ -23,9 +23,35 @@ export class PacienteListarComponent implements OnInit {
 
   ngOnInit(): void {
     this.obtenhaPacientes();
+    this.table.responsive;
   }
+
   obtenhaPacientes() {
     this.pessoaService.obtenhaPacientes().subscribe(pacientes => this.pacientes = pacientes);
   }
 
+  deletaPaciente(id) {
+
+    this.confirmationService.confirm({
+      message: 'Deseja realmente excluir o cadastro?',
+      header: 'Exclusão de cadastro',
+      icon: 'pi pi-info-circle',
+
+      accept: () => {
+        this.pessoaService.deletaPaciente(id).subscribe(paciente => { }, err => { console.log('Erro ao deletar paciente') });
+        this.msgs = [{ severity: 'info', summary: 'Concluído', detail: 'Registro Excluido' }];
+        this.confirmationService.close();
+        location.reload;
```

Comment:
Trata o erro direito com o msgs.
Antes de criar uma msgs coloca this.msgs = []; pra resetar.
joga essa mensagem de registro excluido pra dentro do primeiro parametro do subscribe, o reload tb. desse jeito ae se vc tentar excluir e der errado vc vai dar uma msg que excluiu sem ter excluido.